tools/commands-packages-query.js

```diff
@@ -62,6 +63,44 @@ var formatDependencies = function (versionDependencies) {
   return dependencies;
 };
 
+// Convert package exports into a more helpful format for output in
+// 'meteor show'.
+var formatExports = function (rawexports) {
```

Comment:
yeah I dunno.  restructure?  simplify?

Alternatively, maybe the problem here is that we are having trouble describing this thing because it lacks a name; instead you should just make a mini ExportsList class where formatExports is its constructor and exportsToString is a toString (or asString or stringify or whatever we usually call it) method.  I think the more objects have prototypes rather than are just random arrays of other objects the easier to follow...